packages/vscode/utils/get-shell-exit-cmd.ts

```diff
@@ -1,7 +1,23 @@
 import { env } from 'vscode';
 
-export default function getShellExitCommand(code = 0) {
-  const isPowerShell = env.shell.match(/[\\/]pwsh(\.exe)?$/g);
+export default async function getShellExitCommand(code = 0) {
+
+  // env.shell in 1.71 sometimes returns '' unexpectedly.
```

Comment:
I'll approve the pr as is, but as pr vs code docs, this will be undefined when the shell is not detected. I saw some issues on their github, but they say its resolved now, but I see no danger in keeping it